packages/expo-yarn-workspaces/index.js

```diff
@@ -74,6 +40,7 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
     transformer: {
       ...defaultConfig.transformer,
       // Ignore file-relative Babel configurations and apply only the project's
+      // NOTE: This is not used correctly in the upstream: https://github.com/facebook/react-native/blob/753bb2094d95c8eb2152d2a2c1f0b67bbeec36de/packages/react-native-babel-transformer/src/index.js#L81
```

Comment:
I'm not sure this comment is correct. `enableBabelRCLookup` disables Babel's own [babelrc](https://babeljs.io/docs/options#babelrc) option. When that option is on, Babel itself will look up a config file relative to each JS file being compiled. We turn that off since we want to just use one master Babel config.

Reply:
I'd expect the flag to emulate passing `babelrc: false` to the babel function exactly, but this isn't the case since we still resolve the project `.babelrc` manually and pass it to the functions. Imo the exact usage should disable all babelrc resolution, not just most of it.

Reply:
Here's a suggestion with a clearer explanation:
```suggestion
      // NOTE: The Metro transformer still searches for and uses .babelrc and .babelrc.js files:
      // https://github.com/facebook/react-native/blob/753bb2094d95c8eb2152d2a2c1f0b67bbeec36de/packages/react-native-babel-transformer/src/index.js#L81
      // This is in contrast with Babel, which reads only babel.config.json before evaluating its
      // "babelrc" option: https://babeljs.io/docs/options#configfile
```